src/guides/gatewayIntents.md

```diff
@@ -0,0 +1,26 @@
+# Gateway Intents
+This guide will tell you how to use Gateway Intents in BDFD, and what they are for.
```

Comment:
`This guide will tell you` isn't wrong, but `This guide will explain to you` or `In this guide, you will learn` would be better.